snappy_wrappers/wrappers/varfish_annotator/bam_qc/wrapper.py

```diff
@@ -32,6 +36,48 @@
         print(f"{library_name} {identifier}", file=json_tmpf)
         json_tmpf.flush()
 
+    # Extract the coverage data from the alfred qc files.
+    for alfred_qc in snakemake.input.alfred_qc:
+        with gzip.open(alfred_qc, "rt") as inputf:
+            data = json.load(inputf)
```

Comment:
This looks like only the last input file will be read into `data`, is that really intended?

 Or should the next for-loop be indented one level further?

Reply:
True